etl/tests/embedded/test_heating.py

```diff
@@ -0,0 +1,174 @@
+import typing
+import pytest
+from energuide import bilingual
+from energuide import element
+from energuide.exceptions import InvalidEmbeddedDataTypeError
+from energuide.embedded import heating
+
+
+def sample_node(heating_type: str = 'Furnace',
+                equipment_node: typing.Optional[element.Element] = None,
+                specification_node: typing.Optional[element.Element] = None) -> element.Element:
+    specification_node = sample_specifications() if not specification_node else specification_node
+    equipment_node = sample_equipment() if not equipment_node else equipment_node
+
+    data = """
+<HeatingCooling id="18">
+    <Label>Heating/Cooling System</Label>
+    <CoolingSeason>
+        <Start code="1">January</Start>
+        <End code="12">December</End>
+        <Design code="7">July</Design>
+    </CoolingSeason>
+    <Type1>
+        <FansAndPump hasEnergyEfficientMotor="false">
+            <Mode code="1">
+                <English>Auto</English>
+                <French>Auto</French>
+            </Mode>
+            <Power isCalculated="true" low="0" high="145.5" />
+        </FansAndPump>
+    </Type1>
+    <Type2 shadingInF280Cooling="AccountedFor" />
+</HeatingCooling>
+    """
+    node = element.Element.from_string(data)
+    type1_node: element.Element = node.xpath('Type1')[0]
+    heating_node = sample_heating_node(heating_type)
```

Comment:
This seems odd to have down here when the other node creation calls are done at the top of the function

Reply:
At the top, if they are `None` they get set to their default value. Should I move this line up there with them for consistency, although it's slightly different?

Reply:
I'll leave it up to you, if I were writing it I'd put it at the top with the others, but that seems more like an personal preference thing